contrib/babelfishpg_tsql/src/multidb.c

```diff
@@ -924,6 +924,41 @@ rewrite_column_refs(ColumnRef *cref)
 static void
 rewrite_rangevar(RangeVar *rv)
 {
```

Comment:
rewrite_rangevar is used for all the statements, not just SELECT statements. Please verify the behaviour for other statements as well.